client/src/components/AddCohort/index.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import './style.css';
+import { Form, Input, Button, message } from 'antd';
+import postData from '../postData';
+
+const Demo = () => {
+  const onFinish = async (values) => {
+    const response = await postData('/api/v1/cohorts', values);
+    const { message: resMessage } = response.data.data;
```

Comment:
If there was an error, `response.data` might be undefined, so you might have to move this const into the if-statement to make sure it was a successful post.